github/ci/prow-deploy/kustom/overlays/workloads-production/resources/priority-classes.yaml

```diff
@@ -19,6 +19,15 @@ description: "Allows gpu jobs to be scheduled with higher priority."
 ---
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
+metadata:
+  name: windows
+value: 1000000
```

Comment:
Right, even lower than the place holder actually, if the windows can run on those machine
good catch

Reply:
It raising the follow question - does the vgpu can run on sriov machines ?
if so it might catch the place of sriov-place-holder as well ?
(and if so lets change it on a follow PR)